packages/experiments/src/components/Button/SplitButton/SplitButton.types.tsx

```diff
@@ -61,12 +61,7 @@ export interface ISplitButtonSlots extends IMenuButtonSlots {
 /**
  * {@docCategory Button}
  */
-export interface ISplitButton {
-  /**
-   * Sets focus to the first focus stop of the SplitButton.
-   */
-  focus: () => void;
-}
+export interface ISplitButton extends IMenuButton {}
```

Comment:
Is there still a need for this too?

Reply:
This gets the `focus` method to be a part of the `ISplitButton` interface while still allowing us a place where we can extend this interface for `SplitButton`-only scenarios in the future.